jboss_wildfly/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "display_name": "JBoss/Wildfly",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "jboss_wildfly",
+  "metric_prefix": "jboss.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Gathers various JMX metrics from JBoss and Wildfly Applications",
+  "guid": "ff99b3d2-9c14-4cdf-b869-7b8b1cbf0716",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-JBoss/Wildfly Integration",
+  "categories": [
+    ""
```

Comment:
There has to be at least one category here.

Reply:
Done, thnaks.